packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.js

```diff
@@ -140,14 +140,14 @@ export default class QuanticFacet extends LightningElement {
 
   get facetSearchResults() {
     const results = this.facet.state.facetSearch.values;
```

Comment:
`hasSearchResults` would also fail when `this.facet.state.facetSearch.values` is `null`.